packages/binding-http/test/http-client-oauth-tests.ts

```diff
@@ -64,16 +64,16 @@ class HttpClientOAuthTest {
         });
     }
 
-    static after() {
-        return promisify(this.server.close.bind(this.server))();
+    static async after() {
+        return await promisify(this.server.close.bind(this.server))();
```

Comment:
This line still uses `promisify` where the others use the `new Promise` pattern. I'll open an issue to keep track of this we should use one pattern consistently (personally I found `promisify` less verbose and more explicit)